FileBrowser/src/base-file.js

```diff
@@ -177,20 +177,39 @@ const dragSource = {
   },
 
   endDrag(props, monitor, component) {
-
+    
     if (!monitor.didDrop())
       return;
 
     const item = monitor.getItem();
     const dropResult = monitor.getDropResult();
 
     var fileNameParts = props.fileKey.split('/');
+
     var fileName = fileNameParts[fileNameParts.length - 1];
-    var newKey = `${dropResult.path ? dropResult.path + '/' : ''}${fileName}`;
 
-    if ((dropResult.newKey != props.fileKey) && props.browserProps.renameFile) {
-      props.browserProps.openFolder(dropResult.path + '/');
-      props.browserProps.renameFile(props.fileKey, dropResult.newKey);
+    let pathArray = dropResult.fileKey.split('/')
+    pathArray.pop()
+    let path = pathArray.join('/')
+
+    var newKey = `${path ? path + '/' : '' }${fileName}`;
+
+    let newKeyArray = dropResult.newKey.split('/')
+    let fileKeyArray = props.fileKey.split('/')
+
+    newKeyArray.pop()
+    fileKeyArray.pop()
+
+    let newKeyPath = newKeyArray.join('/')
+    let fileKeyPath= fileKeyArray.join('/')
+
+    newKeyPath = newKeyPath.replace(/\/\/\/g/, '/')
```

Comment:
can you clarify what this line is doing?

Reply:
this is a check that removes any double // in the path. I can push and update and clean this up. it was causing a lot of problems fixing some labmanager-ui file browser bugs.

Reply:
ok got it